packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -308,7 +314,9 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
         RelayData storage relay = relays[depositHash];
         bytes32 instantRelayHash = keccak256(abi.encode(depositHash, relay.realizedLpFeePct));
         require(
-            relays[depositHash].relayState == RelayState.Pending && instantRelays[instantRelayHash] == address(0),
+            (relays[depositHash].relayState != RelayState.Uninitialized ||
```

Comment:
it might be good to add a comment to each of these requires explaining why they are needed, even if they are inline with each. for example:

```
                instantRelays[instantRelayHash] == address(0), // instantRelayer must be zero address (i.e no previous instant relayer)
```

or something to this effect for each require